docs/data/charts/animation/animation.md

```diff
@@ -5,71 +5,79 @@ productId: x-charts
 
 # Charts - Animation
 
-<p class="description">Animate charts for a better look and feel.</p>
+<p class="description">Learn how to customize both CSS and JavaScript-based Chart animations.</p>
 
-Some elements of charts are animated, such as the bars in a bar chart or the slices in a pie chart.
+Some elements of the MUI X Charts are animated by default—for example, the bars in a Bar Chart rise from the axis, and the slices in a Pie Chart expand to fill the circle.
 
-Charts use CSS animations when possible, but some animations can't be done using CSS only. In those cases, JavaScript is used to animate elements.
+These animations are primarily built with CSS, but some use JavaScript-based React hooks as well.
+You can use these hooks to animate other elements of the Charts that aren't animated by default, or to add animations to your own custom components.
 
-The animations of elements that are animated using CSS can be customized by overriding the CSS classes:
+To customize Chart animations, you may need to override CSS classes or implement the custom hooks provided, depending on your specific use case.
+
+## Customizing CSS animations
+
+You can override the default CSS classes to customize CSS-based animations.
+The demo below shows how to do this by adding an animated color gradient to the label text:
```

Comment:
This animation example is weird. It only shows how to add a new animation to an element, rather than updating one of our animations.

@bernardobelchior WDYT about replacing it with something like, changing the timing of the slice animation?

Reply:
This is replacing the default animation of the pie arc label with a custom one. Isn't that what you mean by updating?

Reply:
Any suggestion? I'm open to improving it!

Reply:
Something like [this](https://github.com/mui/mui-x/pull/19185)?